TransferMethod/Sources/View/Widget/AbstractWidget.swift

```diff
@@ -31,17 +31,16 @@ class AbstractWidget: UIStackView, UITextFieldDelegate {
         label.textColor = Theme.Label.textColor
         label.font = Theme.Label.bodyFont
         label.translatesAutoresizingMaskIntoConstraints = false
-        label.widthAnchor.constraint(equalToConstant: 144.0).isActive = true
-        label.setContentHuggingPriority(UILayoutPriority.required, for: NSLayoutConstraint.Axis.horizontal)
+        label.setContentHuggingPriority(UILayoutPriority.defaultHigh, for: NSLayoutConstraint.Axis.horizontal)
         return label
     }()
 
     required init(field: HyperwalletField) {
         super.init(frame: CGRect())
         translatesAutoresizingMaskIntoConstraints = false
         axis = .horizontal
-        alignment = .center
         spacing = 5
+        distribution = .fillEqually
```

Comment:
![image](https://user-images.githubusercontent.com/29735602/62148099-e3a77680-b2ad-11e9-8cb7-d4925d00ddbb.png)

if you see the existing behavior, the spacing is not equal. Label takes less pace than text field. Kindly keep it as it is now instead of assigning equal space to both label and text field

Reply:
Fixed the alignment issue